status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.114.3",
-    "commit-sha1": "7662cd613a9d8bdda4927e583c429b362e3d3815",
-    "src-sha256": "1ldlwhpwdwjxkx9imjinzcw3x5a6wcs70w13kypsfaf4hv17ipmb"
+    "version": "3890748c097b90efdda28f450c2161c4ce963094",
```

Comment:
FYI: As soon as this PR is approved, this file is going to change to point to the actual release number used by the unmerged PR in status-go https://github.com/status-im/status-go/pull/2957